WalletWasabi.Fluent/Views/SettingsPageView.axaml

```diff
@@ -5,7 +5,7 @@
              mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
              x:Class="WalletWasabi.Fluent.Views.SettingsPageView">
   <StackPanel Spacing="10">
-    <TextBlock Text="Settings" FontSize="22" VerticalAlignment="Center" HorizontalAlignment="Center" />
+    <TextBlock Classes="vcenter hcenter h3" Text="Settings" />
```

Comment:
vcenter / hcenter get rid of... this is obsolete line anyway when we merge master and resolve conflicts.